opencensus/stats/exporters/prometheus_exporter.py

```diff
@@ -0,0 +1,339 @@
+                                         documentation=desc['documentation'],
+                                         buckets=list(points.items()),
+                                         sum_value=agg_data.sum,
+                                         labels=labels)
+
+        elif isinstance(agg_data,
+                        aggregation_data_module.SumAggregationDataFloat):
+            labels = desc['labels'] if agg_data.sum_data is None else None
+            return UntypedMetricFamily(name=desc['name'],
+                                       documentation=desc['documentation'],
+                                       value=agg_data.sum_data,
+                                       labels=labels)
+
+        elif isinstance(agg_data,
+                        aggregation_data_module.LastValueAggregationData):
+            labels = desc['labels'] if agg_data.value is None else None
+            return GaugeMetricFamily(name=desc['name'],
+                                     documentation=desc['documentation'],
+                                     value=agg_data.value,
+                                     labels=labels)
+
+        else:
+            raise ValueError("unsupported aggregation type %s"
+                             % type(agg_data))
+
+    def collect(self):  # pragma: NO COVER
+        """	Collect fetches the statistics from OpenCensus
+        and delivers them as Prometheus Metrics.
+        Collect is invoked everytime a prometheus.Gatherer is run
+        for example when the HTTP endpoint is invoked by Prometheus.
+        """
+        for v_data in list(self.view_data):
+            signature = view_signature(self.options.namespace,
+                                       self.view_data[v_data].view)
+            desc = self.registered_views[signature]
+            metric = self.to_metric(desc,
+                                    self.view_data[v_data].view)
+            yield metric
+
+    def describe(self):
+        """ describe will be used by Prometheus Client
+        to retrieve all registered views.
+        """
+        registered = {}
+        for sign in self.registered_views:
+            registered[sign] = self.registered_views[sign]
+
+        for v_data in list(self.view_data):  # pragma: NO COVER
+            signature = view_signature(self.options.namespace, v_data.view)
+            desc = self.registered_views[signature]
+            metric = self.to_metric(desc,
+                                    self.view_data[v_data].view)
+            yield metric
+
+
+class PrometheusStatsExporter(base.StatsExporter):
+    """ Exporter exports stats to Prometheus, users need
+        to register the exporter as an HTTP Handler to be
+        able to export.
+    """
+    def __init__(self,
+                 options,
+                 gatherer,
+                 transport=sync.SyncTransport,
+                 collector=Collector()):
+        self._options = options
```

Comment:
Looking at the `Options` class, all of its args have default value and are optional. So can we also make the `options` param here optional?
```
if options is None:
    options = Options()
```